jdaviz/configs/default/plugins/export/export.py

```diff
@@ -163,6 +185,27 @@ def _sync_singleselect(self, event):
                      'table_selected', 'plot_selected'):
             if name != attr:
                 setattr(self, attr, '')
+            if attr == 'subset_selected':
+                self._set_subset_not_supported_msg()
+
+    def _set_subset_not_supported_msg(self, msg=None):
+        """
+        Check if selected subset is spectral or composite, and warn and
+        disable Export button until these are supported.
+        """
+
+        if self.subset.selected is not None:
+            subset = self.app.get_subsets(self.subset.selected)
+            if self.app._is_subset_spectral(subset[0]):
+                self.subset_invalid_msg = 'Export for spectral subsets not supported.'
+            elif len(subset) > 1:
+                self.subset_invalid_msg = 'Export for composite subsets not supported.'
+            elif self.subset.selected == '':
+                self.subset_invalid_msg = ''
+            else:
+                self.subset_invalid_msg = ''
+        else:  # subset is selected, but valid

```

Comment:
this is actually no subset selected, right?

Reply:
yes, fixed comment to clarify.